HOWTO.md

```diff
@@ -76,31 +76,41 @@ Oh boy, so you've decided to release a new version of libsbp.
     # Produces most recent tag (e.g., v0.29)
     git describe --abbrev=0 --tags
     # Increment that value, create a new one (e.g, v0.30), and push
-    git tag -a INCREMENTED_TAG
-    git push upstream INCREMENTED_TAG
+    git tag -a INCREMENTED_TAG -m "Version INCREMENTED_TAG of libsbp."
+    git push origin INCREMENTED_TAG
     ```
 
 1. Verify that package dependencies, their version numbers, and the
    libsbp version number in the C, Python, JavaScript, and LaTeX developer
    documentation are consistent.
 
 2. Add to RELEASE_NOTES.md and update the CHANGELOG details with `make
-   release`. Submit a pull request. This requires
+   release`. Submit a pull request and get it merged. This requires
    [github-changelog-generator](https://github.com/skywinder/github-changelog-generator),
-   and a CHANGELOG_GITHUB_TOKEN in your PATH if you don't already have
+   and a `CHANGELOG_GITHUB_TOKEN` in your `PATH` if you don't already have
    them.
 
-3. Create a release on
+3. After the release PR is merged, recreate the tag:
+    ```shell
+    git checkout master
+    git pull
+    git tag -d INCREMENTED_TAG
+    git tag -a INCREMENTED_TAG -m "Version INCREMENTED_TAG of libsbp."
+    git push --force origin INCREMENTED_TAG
```

Comment:
don't push tags at all until the final one - they don't have to be on remote for any of these steps